controllers/grafanafolder_controller.go

```diff
@@ -178,11 +178,12 @@ func (r *GrafanaFolderReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 			}
 			return ctrl.Result{}, nil
 		}
-		controllerLog.Error(err, "error getting grafana folder cr")
-		return ctrl.Result{RequeueAfter: RequeueDelay}, err
+		return ctrl.Result{RequeueAfter: RequeueDelay}, fmt.Errorf("error getting grafana folder cr: %w", err)
```

Comment:
since we're already touching this, can you remove the `RequeueAfter` from the `ctrl.Result`? If an error is returned, the delay is ignored and results in an annoying warning